src/Runner.ts

```diff
@@ -2,15 +2,17 @@ import { ProjectFileType } from '@dojo/cli-export-project/interfaces/project.jso
 import Evented from '@dojo/core/Evented';
 import { createHandle } from '@dojo/core/lang';
 import project from './project';
+import * as base64 from './support/base64';
```

Comment:
Does this belong as a support helper, or something in core, or revisit doing something with crypto? Not a blocking comment, just something we should open an issue for elsewhere?